web/src/components/storage/utils.js

```diff
@@ -149,6 +149,36 @@ const hasFS = (volume, fs) => {
   return volFS.toLowerCase() === fs.toLocaleLowerCase();
 };
 
+/**
+ * Checks whether the given volume has snapshots.
+ *
+ * @param {import(~/clients/storage).Volume} volume
```

Comment:
NP: I'd move these type imports to the very top of the file instead of _importing_ them in each @param they are needed.